metasv/run_spades.py

```diff
@@ -170,11 +165,16 @@ def run_spades_parallel(bam=None, spades=None, spades_options="", bed=None, work
         return None, None
 
     bedtool = pybedtools.BedTool(bed)
-    total = bedtool.count()
 
     chrs = set(chrs)
     all_intervals = [interval for interval in bedtool] if not chrs else [interval for interval in bedtool if
                                                                          interval.chrom in chrs]
+    if slicing is not None:
```

Comment:
Since None evaluates to false and you also want to skip slicing = [], you can simply have `if slicing:`